### PR TITLE
Create an example OpenAPI spec

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Created a unified OpenAPI service spec example with `mutation`, `validation` and `metadata` endpoints.
- Added the reference to it to the ClusterDocs topic for the Asset Store to render it on the website
- Added reference to the new example in Asset Store docs.
- Removed `metadata-service.yaml`, `mutation-service.yaml`, and `validation-service.yaml` and references to them in docs.

**Related issue(s)**
See also #5973

### CODE REVIEW

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          multipart/form-data:
+            schema:
+              type: object
+              additionalProperties:
+                type: string
+                format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successResponse"
+servers:
```
Comment (derberg) [inline docs/asset-store/assets/example-openapi-service.yaml]: why do we need it, why not just baseURL?

**Comment by derberg** [comment]
either unify the file as I wrote, with extract endpoint, and make it clear in main description of the file, that example is based on a case of Asset resources that contain asyncapi

or lorem ipsum like @magicmatatjahu wrote

**Comment by derberg** [comment]
also please, in description of the endpoints, map them to the names of the webhooks, so it is clear that `convert` is for mutationWebhookService and `extract` for matedataWebhookService or rename them mutation and metadata and really focus on lorem ipsum

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 1 above)
Comment (magicmatatjahu) [reply docs/asset-store/assets/example-openapi-service.yaml]: We discuss with @kazydek to remove it from spec :)

**Comment by derberg** [comment]
@kazydek sorry, but I'm now playing with **spectral** at a conference and seems like `servers should actually stay, sorry. Please also add [contact object](https://swagger.io/specification/#contactObject) 

```
Desktop $spectral lint example-openapi-service.yaml
OpenAPI 3.x detected

/Users/i303812/Desktop/example-openapi-service.yaml
 1:1  warning  api-servers   OpenAPI `servers` must be present and non-empty array.
 2:6  warning  info-contact  Info object should contain `contact` object.

✖ 2 problems (0 errors, 2 warnings, 0 infos, 0 hints)
```

for contact I suggest. you need to place it inside info
```
  contact:
    name: The Kyma project authors
    url: http://slack.kyma-project.io
    email: kyma-sig-core@googlegroups.com
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
+openapi: 3.0.0
+info:
+  description: This example shows API specification of a service that exposes endpoints for all Asset Store webhooks. It contains `/convert`, `/validate`, and `/extract` endpoints.
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
  description: This example shows an API specification of a service that exposes endpoints for all Asset Store webhooks. It contains the `/convert`, `/validate`, and `/extract` endpoints.
```

File: docs/asset-store/03-04-supported-webhooks.md [hunk 3]
```diff
@@ -25,19 +25,15 @@ If you create a specific mutation, validation, or metadata service for the avail
...
 
-See the example of the validation service with the `/validate` endpoint [here](./assets/validation-service.yaml).
-
 - **metadata service** must expose endpoints that:
 
   - pass file data in the `"object": "string"` format in the request body, where **object** stands for the file name and **string** is the file content.
   - return the `200` response with extracted metadata.
 
-See the example of the validation service with the `/extract` endpoint [here](./assets/metadata-service.yaml).
+See the example of a service with the `/convert`, `/validate`, and `/extract` endpoints [here](./assets/example-openapi-service.yaml).
```
Comment (mmitoraj) [inline docs/asset-store/03-04-supported-webhooks.md]: is is an example of a service or of a specification?
```suggestion
See the example of an API specification with the `/convert`, `/validate`, and `/extract` endpoints [here](./assets/example-openapi-service.yaml).
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successResponse"
+servers:
+  - url: http://localhost:3000/v1
+components:
+  responses:
+    successConversionResponse:
+      description: Successful response with the new file content
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
      description: A successful response with the new file content
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: "{\"info\":{\"version\":\"1.0.0\",\"title\":\"AnyOf example\"},\"topics\":{\"test\":{\"publish\":{\"$ref\":\"#\/components\/messages\/testMessages\"}}},\"asyncapi\":\"1.1.0\",\"components\":{\"messages\":{\"testMessages\":{\"payload\":{\"anyOf\":[{\"$ref\":\"#\/components\/schemas\/objectWithKey\"},{\"$ref\":\"#\/components\/schemas\/objectWithKey2\"}]}}},\"schemas\":{\"objectWithKey2\":{\"type\":\"object\",\"properties\":{\"key2\":{\"type\":\"string\"}}},\"objectWithKey\":{\"type\":\"object\",\"properties\":{\"key\":{\"type\":\"string\"}}}}}}"
+    notModified:
+      description: File content was not modified
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
      description: File content was not modified.
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    successValidationResponse:
+      description: Successful response
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
      description: A successful response
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+                  $ref: "#/components/schemas/ResultSuccess"
+    unprocessableEntity:
+      description: Unprocessable entities. File validation failed.
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: Invalid property
+    successResponse:
+      description: Successful response
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
      description: A successful response
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+                type: array
+                items:
+                  $ref: "#/components/schemas/ResultSuccess"
+    unprocessableEntity:
+      description: Unprocessable entities. File validation failed.
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: Invalid property
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
            example: An invalid property
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+    successResponse:
+      description: Successful response
+  schemas:
+    ResultSuccess:
+      description: "It represents successful processing of a single file."
+      type: object
+      properties:
+        filePath:
+          description: "Relative or absolute path to the file which was sent in the form body"
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
          description: A relative or an absolute path to the file which was sent in the form body
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: Invalid property
+    successResponse:
+      description: Successful response
+  schemas:
+    ResultSuccess:
+      description: "It represents successful processing of a single file."
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
      description: It represents successful processing of a single file.
```

File: docs/asset-store/assets/example-openapi-service.yaml [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+      type: object
+      properties:
+        filePath:
+          description: "Relative or absolute path to the file which was sent in the form body"
+          type: string
+        metadata:
+          description: "Front matter metadata extracted from the file"
```
Comment (mmitoraj) [inline docs/asset-store/assets/example-openapi-service.yaml]:
```suggestion
          description: Front matter metadata extracted from the file
```

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 4 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: As discussed, we are omitting the articles at the beginning of response descriptions.

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 6 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: as above

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 5 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: as agreed `description: File content not modified`

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 7 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: same here

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 8 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: and here

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 9 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: (but without articles at the beginning)

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 10 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: I changed it into `description: Successful processing of a single file`

File: docs/asset-store/assets/example-openapi-service.yaml (hunk 1 above)
Comment (kazydek) [reply docs/asset-store/assets/example-openapi-service.yaml]: @derberg If I were to change `url` into `baseURL` what link would you provide here?

Comment (derberg) [reply docs/asset-store/assets/example-openapi-service.yaml]: sorry, I introduced some confusion here, sorry for that
baseUrl (actually host and basePath) is a thingy for swagger 2.0 
in 3.0 it is solved totally differently https://swagger.io/specification/#server-object-example
so what you have now is good, I would not change that, sorry again

**Pull request merged**